cmake/therock_subproject.cmake

```diff
@@ -777,7 +777,7 @@ function(therock_cmake_subproject_activate target_name)
     add_custom_command(
       OUTPUT "${_stage_stamp_file}"
       # Populate local dist directory with this+all transitive stage installs.
-      COMMAND "${Python3_EXECUTABLE}" "${_fileset_tool}" copy "${_dist_dir}" ${_dist_source_dirs}
+      COMMAND "${Python3_EXECUTABLE}" "${_fileset_tool}" copy "${_dist_dir}" ${THEROCK_FILESET_VERBOSE_ARG} ${_dist_source_dirs}
```

Comment:
Can you show an example command line with this? I'm wondering if the positioning of `--verbose` in between `"${_dist_dir}" ${_dist_source_dirs}` looks natural enough. We could also move it ahead of  `"${_dist_dir}"`, especially if `${_dist_source_dirs}` is a list of items and positioning in the argument list matters.

Reply:
I'd say either move it to the front or the back in that case.